backend/app/router/endpoints/users.py

```
@@ -8,6 +8,6 @@
     "bob": {"name": "Bob", "email": "bob@example.com"},
 }
 
-@router.get("/user/{username}")
+@router.get("/")
 def get_user(username: str):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)